.circleci/config.yml

```diff
@@ -16,16 +16,11 @@ jobs:
           command: ./build.sh
       - store_artifacts:
           path: ./site
-      - run:
```

Comment:
Shouldn't we still deploy the bleeding-edge builds from branches etc. somewhere still? Or do you plan on re-introducing it?

Reply:
Yes I will re-introduce it when we got a new machine for that